### PR TITLE
Sort role permissions

### PR DESCRIPTION
**Pull request by AbdelrahmanElawady**
<hr/>
Update Traffic Ops roles GET endpoint to return roles with sorted permissions
## Which Traffic Control components are affected by this PR?

- Traffic Ops

## What is the best way to verify this PR?

- Login to Traffic Portal
- Navigate to `User Admin` -> `Roles`
- Check any role

## PR submission checklist
- [x] This PR has tests 
- [ ] This PR has documentation 
- [ ] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

Closes: #7403

### CODE REVIEW

**Comment by ocket8888** [comment]
Can you please add a description to this PR? As it stands, I have no idea what the intended change is.

**Comment by AbdelrahmanElawady** [comment]
My bad, I updated the description and will add some tests to the new changes.

File: traffic_ops/traffic_ops_golang/role/roles.go [hunk 1]
```diff
@@ -644,6 +645,9 @@ func Get(w http.ResponseWriter, r *http.Request) {
 			api.HandleErr(w, r, tx, http.StatusInternalServerError, nil, fmt.Errorf("scanning RoleV4 row: %w", err))
 			return
 		}
+		sort.Slice(roleV4.Permissions, func(i, j int) bool {
+			return roleV4.Permissions[i] < roleV4.Permissions[j]
+		})
```
Comment (zrhoffman) [inline traffic_ops/traffic_ops_golang/role/roles.go:650]: Using `sort.Strings` instead should work.

**Review by zrhoffman — CHANGES_REQUESTED** [review]
Tests look good, just 1 comment about the sorting.

**Pull request merged**